From d01c693faa76d957a537c3a7edef08553ad438a9 Mon Sep 17 00:00:00 2001
From: Andrew Bayer <andrew@cloudera.com>
Date: Thu, 22 Dec 2011 11:27:05 -0800
Subject: [PATCH 030/795] MR1: Handle commons-daemon POM inconsistency.
 (cherry picked from commit eccce0e8d4981646a61245bb0aa6652df842f73d)

(cherry picked from commit 9d679d34d389c2af5cc9a91ece91b0add0245a68)
---
 ivy/ivysettings.xml |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/ivy/ivysettings.xml b/ivy/ivysettings.xml
index 90f88fe..d741079 100644
--- a/ivy/ivysettings.xml
+++ b/ivy/ivysettings.xml
@@ -61,6 +61,7 @@
       root="${repo.maven.org}"
       pattern="${maven2.pattern.ext}"
       m2compatible="true"
+      checkconsistency="false"
       />
     <ibiblio name="apache-snapshot"
       root="${snapshot.apache.org}"
-- 
1.7.0.4

